dozer-api/src/grpc/shared_impl/mod.rs

```diff
@@ -97,6 +97,10 @@ pub fn on_event<T: Send + 'static>(
         ));
     }
 
+    if endpoints.is_empty() {
+        return Err(Status::invalid_argument("empty-endpoints-array"));
```

Comment:
Don't add `-`s.

Reply:
Spaces are not handled properly in the output. It is shown like this in output: `empty%20endpoints%20array`. Is that fine?

Reply:
I believe it's frontend problem that the message is not properly displayed. Try postman?

Reply:
Right. Its a frontend issue.